docs/src/Mesh.md

```diff
@@ -54,3 +54,38 @@ Filters.CutoffFilter
 Filters.ExponentialFilter
 Filters.TMARFilter
 ```
+
+## `Interpolation`
+
+### Constructors
+
+```@docs
+Interpolation.InterpolationBrick(grid::DiscontinuousSpectralElementGrid{FT},
+                       xbnd::Array{FT,2},
+                        x1g::AbstractArray{FT,1},
+                        x2g::AbstractArray{FT,1},
+                        x3g::AbstractArray{FT,1}) where FT <: AbstractFloat
```

Comment:
```suggestion
Interpolation.InterpolationBrick
```
You don't need the full signature, you can just use the function name (same for all below)